rd_ui/app/views/visualizations/map_editor.html

```diff
@@ -0,0 +1,55 @@
+<div class="form-horizontal">
+    <div class="form-group">
+        <label class="col-lg-2">Map height (px)</label>
+        <div class="col-sm-4">
+            <input class="form-control" type="text" ng-model = "visualization.options.height" />
```

Comment:
Better to have here `type="number"` (then it's being saved as number, and you can control the value with arrows).